test/sequential/test-debugger-debug-brk.js

```diff
@@ -1,10 +1,13 @@
 'use strict';
 const common = require('../common');
 common.skipIfInspectorDisabled();
+const fixtures = require('../common/fixtures');
 const assert = require('assert');
 const spawn = require('child_process').spawn;
 
-const script = `${common.fixturesDir}/empty.js`;
+// file name here doesn't actually matter since
+// debugger will connect regardless of file name arg
```

Comment:
I was confused why the test was passing regardless of the file name so I added this comment. Talked with @Trott about this. Happy to remove if this is common knowledge!